librz/cons/histogram.c

```diff
@@ -368,13 +374,19 @@ RZ_API RZ_OWN RzStrBuf *rz_histogram_interactive_horizontal(RZ_NONNULL RzHistogr
 		for (j = 0; j < width; j++) {
 			int realj = adder + (j)*histogramwidth / (zoom * width);
 			int realjnext = adder + (j + 1) * histogramwidth / (zoom * width);
+			unsigned char curdata = 0;
 			if (sizeofonebar > 1) {
 				realj = adder + j;
+				curdata = data[realj];
 			}
 			if (sizeofonebar == 1 && realj <= hist->barnumber && realjnext > hist->barnumber) {
 				realj = hist->barnumber;
+			} else if (sizeofonebar == 1) {
+				for (int i = realj; i < realjnext; i++)
+					curdata += data[i];
```

Comment:
brackets. also this snippet of code looks copied and pasted in 2 different places, maybe extract this in a `static` function? so we avoid code duplication?